packages/paste-website/src/utils/GraphqlUtils.ts

```diff
@@ -63,12 +63,8 @@ const getMutation = (key: string): MutationFunction => {
     case 'allAirtable': {
       return ({status, Figma, ...rest}) => ({
         packageStatus: sentenceCase(status),
-        figmaStatus: Figma === null ? undefined : Figma,
```

Comment:
We don't need the null check now?

Reply:
No, the packageStatusLegend checks if it's null or not now. This is what caused the problem.